test/functional/test-custom-element.js

```diff
@@ -589,18 +589,32 @@ describe('CustomElement', () => {
     expect(element2.sizerElement_.style.paddingTop).to.equal('1%');
   });
 
-  it('should change height without sizer', () => {
+  it('should change size without sizer', () => {
     const element = new ElementClass();
-    element.changeHeight(111);
+    element.changeSize(111, 222);
     expect(element.style.height).to.equal('111px');
+    expect(element.style.width).to.equal('222px');
   });
 
-  it('should change height with sizer', () => {
+  it('should change size - height only without sizer', () => {
+    const element = new ElementClass();
+    element.changeSize(111);
+    expect(element.style.height).to.equal('111px');
+  });
+
+  it('should change size - width only without sizer', () => {
+    const element = new ElementClass();
+    element.changeSize(undefined, 111);
+    expect(element.style.width).to.equal('111px');
+  });
+
+  it('should change size with sizer', () => {
     const element = new ElementClass();
     element.sizerElement_ = document.createElement('div');
-    element.changeHeight(111);
+    element.changeSize(111, 222);
```

Comment:
Let's add one more test where width and height are `0` since we had a bug in that part.